Sources/Kommunicate/Classes/KMPushNotificationHelper.swift

```diff
@@ -148,6 +148,8 @@ public class KMPushNotificationHelper {
         case "FaqViewController",
              "RatingViewController":
             return true
+        case configuration.embeddedVCName:
```

Comment:
Why keep this logic in `ALKConfiguration`? We can keep it here in KM. Also, we can store the name ourselves based on the root view they share in launch function.

Reply:
we can't keep it in KMConfiguration. its an extension file & we can't store values there. thatswhy I kept it in ALKConfiguration. We can internally get the value of the virecontroller name. That's done

Reply:
If we are setting it ourselves, then why add it in the configuration? We can keep it in userdefaults or static var.

Reply:
removed